src/sentry/receivers/onboarding.py

```diff
@@ -20,6 +24,23 @@
 from sentry.utils.javascript import has_sourcemap
 
 
+def check_for_onboarding_complete(organization):
```

Comment:
@dcramer Do you think it's worth doing this now? I'm concerned because even if a project doesn't have onboarding enabled, this gets run sometimes a few times per event.